cmd/swctl/app/cmd_dependency.go

```diff
@@ -218,35 +274,36 @@ func linkSetUpDown(cli Cli) *cobra.Command {
 	return cmd
 }
 
-func (*Dependencies) IsDockerAvailable(cli Cli) bool {
+func IsDockerAvailable(cli Cli) (bool, error) {
 	out, _, err := cli.Exec("whereis docker", nil)
 	if err != nil {
-		panic(err)
+		return false, err
 	}
 	if strings.Contains(out, "/docker") {
-		return true
+		return true, nil
 	}
-	return false
+	return false, nil
 }
 
-func (*Dependencies) IsHugePagesEnabled(cli Cli) (bool, int) {
+func AllocatedHugePages(cli Cli) (int, error) {
 	out, _, err := cli.Exec("sysctl vm.nr_hugepages -n", nil)
 	if err != nil {
-		panic(err)
+		return 0, err
 	}
 	hugePgSize, err := strconv.Atoi(strings.TrimSpace(out))
 	if err != nil {
-		fmt.Println(err)
+		return 0, err
 	}
 	if hugePgSize == 0 {
-		return false, hugePgSize
+		return 0, err
 	}
 
-	return true, hugePgSize
+	return hugePgSize, err
 }
 
-func (*Dependencies) ResizeHugePages(cli Cli, size uint) error {
-	//TODO: Handle numa case, Big hugepages(are immutable and can be setted only during booting)
+func ResizeHugePages(cli Cli, size uint) error {
+	//TODO: Make persistent hugepages
+	//TODO: Handle numa case, Big (1GB)hugepages(are immutable and can be setted only during booting)
 	if size == 0 {
 		fmt.Fprintln(cli.Out(), "Skipping hugepages")
```

Comment:
This looks superior to fmt.Print(....) that is used in other places. Can you replace it everywhere?